docroot/themes/humsci/humsci_basic/src/scss/airy.scss

```diff
@@ -0,0 +1,3 @@
+$current-theme: 'airy';
```

Comment:
I'd like to play around with CSS variables a bit too to see if they give us anything else

Reply:
defer this to next css work